packages/tools/eslint-config/profile/lib.js

```diff
@@ -1,3 +1,10 @@
+const fs = require('fs');
+const path = require('path');
+
+const prettierOptions = JSON.parse(
+  fs.readFileSync(path.resolve(__dirname, '../../../../.prettierrc'), 'utf8'),
+);
+
```

Comment:
Can we add a comment that this is specific to our monorepo where, according to `rush.json#projectFolderMin/MaxDepth`, a package will **always** be in the 3rd nested directory (`packages/{apps,libs,tools}/package-name/*`). 

Should we also add to the comment that a more generic alternative could be to use:

```js
const prettierrcPath = `${require('child_process')
  .execSync('git rev-parse --show-toplevel')
  .toString()
  .replace(/\n/, '')}/.prettierrc`;
```

Reply:
done